packages/admin-ui/ui/src/components/organisms/product-variants-section/edit-variant-inventory-modal.tsx

```diff
@@ -167,15 +167,17 @@ const EditVariantInventoryModal = ({ onClose, product, variant }: Props) => {
     // @ts-ignore
     onUpdateVariant(
       variant.id,
-      removeNullish({
-        ...dimensions,
-        ...customs,
-        ...stock,
-        ean,
-        barcode,
-        upc,
-        allow_backorder,
-      }),
+      {
+        ...removeNullish({
```

Comment:
todo: Should we update removeNullish instead to check !`= null or != ''` intead in order to prevent stripping false values? wdyt?

Reply:
renaming existing utility to falsy and updating remove nullish to be `== null` since that evaluates to true for `null` and `undefined` (source: https://dorey.github.io/JavaScript-Equality-Table/)